Drift/Private/DriftSchemas.h

```diff
@@ -609,18 +609,18 @@ struct FDriftFriendResponse
 
 struct FDriftFriendRequestsResponse
 {
-	FString accept_url;
+	FString accept_url = "";
 	FDateTime create_date;
 	FDateTime expiry_date;
-	int32 id;
-	FString issued_by_player_name;
-	int32 issued_by_player_id;
-	FString issued_by_player_url;
-	int32 issued_to_player_id;
-	FString issued_to_player_name;
-	FString issued_to_player_url;
+	int32 id = 0;
+	FString issued_by_player_name = "";
```

Comment:
I don't recall it being required to initialize FStrings, if doing so, one should use the `TEXT("")` macro, always.

Reply:
I don't know about that, but I was getting strange values for the integers so I initialized the FStrings as well just in case.

Reply:
The string holds the data in a `TArray<TCHAR>` which is empty by default so I think starting to randomly initialize it will just be confusing

Reply:
Turns out FStrings don't need to be explicitly initialized this way due to their default constructor.
Removed the initialization for FStrings.